woocommerce/woocommerce
# CYS: Show Patterns from PTK

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR shows the patterns fetched from the PTK source. The user clicking on them can add in the Site.

Closes #47461 #47463 #47465 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure that you have the last version of Gutenberg enabled
2. Make sure you have the WooCommerce beta tester plugin installed
3. Head over to Tools > WCA Test Helper > Features
4. Make sure you see the `pattern-toolkit-full-composability` on the list.
5. Enable it.
6. Head over to WooCommerce > Home > Customize your store.
7. Click on Design Your Homepage.
8. Click a category.
9. Click on different patterns.
10. Ensure that the patterns are added to the Site.
11. Click on a pattern.
12. Ensure that the tooltip is under the mouse pointer.
13. Save.
14. Ensure that the patterns are visible on the frontend.


<video src="https://github.com/woocommerce/woocommerce/assets/4463174/6e06aa2a-c2de-428e-9f94-901de7d9d04f" />





### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Show Patterns from PTK

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
